src/hooks/screen/useChannelHook.ts

```diff
@@ -3,41 +3,45 @@ import {PostNotificationChannelList} from '../../../types/database/schema/PostNo
 
 function useChannelHook() {
   const determinePostType = (data: PostNotificationChannelList): BaseChannelItemTypeProps => {
-    const isOwningPost = data?.rawJson?.isOwnPost;
+    const isOwnSignedPost =
+      data?.rawJson?.isOwnSignedPost && data?.channelType === 'POST_NOTIFICATION';
+    const isOwnAnonymousPost =
+      data?.rawJson?.isOwnAnonymousPost && data?.channelType === 'ANON_POST_NOTIFICATION';
     const isAnonymousPost = data?.rawJson?.isAnonym;
 
     const firstComment = data?.rawJson?.comments[0];
     const hasComment = data?.rawJson?.comments.length > 0;
     const isOwningComment = firstComment?.reaction?.isOwningReaction;
     const isAnonymousComment = firstComment?.reaction?.data?.is_anonymous;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These lines are checking if the first comment exists and if it belongs to the current user or is anonymous. It's good that you're using optional chaining (`?.`) to prevent possible `undefined` errors when accessing nested properties.



<!-- This is an auto-generated comment by OSS CodeRabbit -->